inspirehep/modules/hal/tei.py

```diff
@@ -0,0 +1,126 @@
+from jinja2 import Environment, PackageLoader
+from nameparser import HumanName
+
+from inspirehep.modules.records.json_ref_loader import replace_refs
+from inspirehep.utils.dedupers import dedupe_list
+from inspirehep.utils.record import get_value
+
+TEMPLATE = 'hal_tei.xml'
```

Comment:
Constants should go to the `config` of the package, and should have more descriptive names. Maybe `HAL_EXPORTER_TEMPLATE`?